### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by wasamasa**
I've found some errors in the reference documentation and went over them with @amirouche on Freenode.

### CODE REVIEW

File: srfi-167.html [hunk 1]
```diff
@@ -157,6 +157,11 @@ <h3 id="engine-unpack"><code>(engine-unpack engine bytevector)</code></h3>
     <p>Call an unpacking procedure that will decode a bytevector
         encoded with the above pack procedure into a Scheme object.</p>
 
+    <h3 id="make-default-engine"><code>(make-default-engine)</code></h3>
+
+    <p>Return an <code>engine</code> record for the current <code>okvs</code>
+        implementation.</p>
+
```
Comment (amirouche) [inline srfi-167.html]: `make-default-engine` is part of the "`okvs`" scheme library, not part of the `engine` library that is why it is documented in the okvs part. It provides the default engine implementation. Where engine is not specific to particular backend, the default engine is.

ref: https://github.com/scheme-requests-for-implementation/srfi-167/pull/9/commits/a6646be7f9a09c88788dfa86229752ace56ca933

**Review by amirouche — CHANGES_REQUESTED** [review]
Please, drop the commit https://github.com/scheme-requests-for-implementation/srfi-167/pull/9/commits/a6646be7f9a09c88788dfa86229752ace56ca933 

Or maybe we should specify that this particular procedure really is part of the okvs backend implementation and not the engine.

**Comment by wasamasa** [comment]
It has been fixed by a later commit, I've combined them now to clear that up.

**Comment by amirouche** [comment]
LGTM!

**Pull request merged**